app/routes/meetings/MeetingDetailRoute.tsx

```diff
@@ -22,7 +26,10 @@ const loadMeeting = (
     },
   },
   dispatch
-) => (loggedIn ? dispatch(fetchMeeting(meetingId)) : Promise.resolve());
+) =>
+  loggedIn
+    ? dispatch(fetchMeeting(meetingId), fetchEmojis())
```

Comment:
Are you sure you can dispatch like this? According to the docs, this is not an option.
You could rather do a `Promise.all([dispatch(fetchmeeting(...)), dispatch(fetchemojis())]`

Reply:
Also, why are you fetching emojis here and passing `fetchEmojis` into the reaction picker?

Reply:
Hmm I'm not entirely sure actually. It was written that way in the ArticleDetailRoute.tsx file, but I'll try the promise.all instead.